core/src/main/scala/pureconfig/backend/TypesafeConfigWrapper.scala

```diff
@@ -0,0 +1,38 @@
+package pureconfig.backend
+
+import java.nio.file.Path
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import pureconfig.error.{ ConfigReaderFailures, ThrowableFailure }
+
+import scala.util.control.NonFatal
+
+/**
+ * A wrapper of [[com.typesafe.config.ConfigFactory]] whose methods return `Either` instead
+ * of throwing exceptions
+ */
+object ConfigFactoryWrapper {
```

Comment:
I prefer having this file named `ConfigFactoryWrapper`, to better reflect its contents.